### PR TITLE
Upgrade AWS, Guava, Gson, Lombok and fix tests

### PR DESCRIPTION
**Pull request by tlehman**
aws-sdk (1.8.7) -> (1.11.228)
google.guava (17.0) -> (23.5-jre)
google.code.gson (2.2.4) -> (2.8.2)
projectlombok (1.14.4) -> (1.16.14)
slf4j (1.7.7) -> (1.7.21)

### CODE REVIEW

File: distribute.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ apply plugin: 'signing'
 sourceCompatibility = 1.8
 
 group = 'com.flightstats'
-version = '0.1.14-SNAPSHOT'
+version = '0.1.15-SNAPSHOT'
```
Comment (luanasmith) [inline distribute.gradle]: I think we can go with version 0.1.15

**Comment by tlehman** [comment]
Semantic versioning rules suggest that backwards incompatible changes should change the minor patch number (I had to fix a test that broken when I upgraded)
https://semver.org/

So 0.2 instead of 0.1.15, and "SNAPSHOT" will be dropped

**Comment by luanasmith** [comment]
These changes look good to me.  I'll leave this open for a few more hours in case anyone else wants to comment.

**Comment by tlehman** [comment]
@luanasmith Sounds good, I am working on some warnings too, so I'll have another commit once I fix those.

**Pull request merged**